lib/foreman_discovery/engine.rb

```diff
@@ -19,6 +19,19 @@ class Engine < ::Rails::Engine
       require_dependency File.expand_path("../../../app/models/setting/discovered.rb", __FILE__) if (Setting.table_exists? rescue(false))
     end
 
+    initializer 'foreman_discovery.assets.precompile' do |app|
+      app.config.assets.precompile += %w(
+        'discovered_hosts.js',
+      )
+    end
+
+    initializer 'foreman_discovery.configure_assets', :group => :assets do
+      SETTINGS[:foreman_discovery] =
+          {:assets => {:precompile => [
+              'discovered_hosts.js',
+          ]}}
+    end
+
```

Comment:
Isn't this an overkill for such a tiny feature? I mean, this needs a change in our packaging. Can you make a change for both Red Hat and Debian packages? https://github.com/theforeman/foreman-packaging